### ISSUE TITLE
Map and vision are gone after Compendium Update 

### ISSUE THREAD

**Issue by destinationbugged** [ValveSoftware/Dota-2#2119]
#### Your system information

* System information: MacBook Pro (13-inch, M1, 2020) 
<img width="2560" alt="Bildschirmfoto 2022-09-02 um 01 26 55" src="https://user-images.githubusercontent.com/112668380/188029888-8b40a0fd-b83f-42ec-9e8a-25d091349cb0.png">

* Have you checked for system updates?: Yes 
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: Yes

#### Please describe your issue in as much detail as possible:
As soon as a match starts, the map is gone and there is no vision at all. This happens with open gl. 
If I use Vulcan, everything shows up but the fps are around 30. 

#### Steps for reproducing this issue:

1. start a match (hero testing, turbo, bots, ranked)

**Comment by DiegoCatalano** [comment]
The same here, I'm using Ubuntu 20.04

In Ubuntu it happens in OpenGL and also in Vulkan

**Comment by stryan** [comment]
I had this until I switched to Vulkan.

> If I use Vulcan, everything shows up but the fps are around 30.

Did you try letting it stabilize? The first time you'll run it you'll have to let it compile shaders.

**Comment by destinationbugged** [comment]
> > If I use Vulcan, everything shows up but the fps are around 30.
> 
> Did you try letting it stabilize? The first time you'll run it you'll have to let it compile shaders.

sure, I finished the game using vulcan and I run a couple of bot games after that. even after 40 min it wasn't better
macOS has its issues with vulcan in general, even before the update I tried to switch sometimes and it wouldn't work out

**Comment by cvlvxi** [comment]
This is also happening to me (osx). As the poster stated -vulkan is just not viable on mac m1. Please fix -gl for osx / linux.

THanks!

**Comment by sskf1990** [comment]
same issue here. Lost a game T.T . Imagine playing 1h without mini map and effects

**Comment by sskf1990** [comment]
is this fixed yet?

**Comment by tanuj101** [comment]
Facing the same issue

**Comment by destinationbugged** [comment]
> is this fixed yet?

nope still the same but Vulcan now gives me out 60 fps, at least playable

**Comment by holydrug** [comment]
Facing the same issue
ARCH linux

**Comment by ToxyFlog1627** [comment]
+1, also arch

**Comment by msaboor35** [comment]
+1, plz fix. ubuntu 22.04.

**Comment by snorlax212** [comment]
+1 ubuntu 22.04. -gl parameter used. vulkan not working on my laptop

**Comment by destinationbugged** [comment]
Vulcan has gotten slightly better with macOS. 
I tried it once again and started watching a game. The fps were at 60 from start on and "pretty stable". 
Also a game with bots gave me +-60. better than nothing. Hope Volvo is working on this issue since I already bought battle points for about 250€ before starting the first game after the update :(

**Comment by AngryPenguinPL** [comment]
Hi. I see same issue on OpenMandriva Cooker. Tested on two GPU one RX580 and second old HD5850 - both affected.
Only Vulkan api works but offer much worse performance on RX580 than OpenGL API.

Also switching for Vulkan API is not a solution for many people. Many linux users still use old GPU that not support Vulkan API, thats why we need good working OpenGL backend I know it's not a priority for Valve anymore. But it is worth taking care of it so that people with older hardware can still play Dota on Linux. I know you were supposed to remove the OpenGL backend from Dota completely but please don't, it's important to a lot of people.

Why I asking to keep OpenGL working:
- Vulkan is not available on many older GPU like my HD5850 that still can handle game.
- Vulkan offer much worse performance than OpenGL (at least on my RX580). Right now Vulkan API is almost not playable.

**Comment by abdullahkhalids** [comment]
Vulkan didn't work on my computer last year, but I tried today and it seems to mostly work. After a couple of games, Neutral Camp boundaries became blurry and faded, and attacks and spells would result in large artifacts.

https://gist.github.com/abdullahkhalids/016402dff664dcd97657b6f654fff882

**Comment by kostandy** [comment]
Having the same issue on my Ubuntu 22.04 (w/ -gl launch option)

![image](https://user-images.githubusercontent.com/12787555/188226814-d4425d76-8f26-4368-bf95-ee24d4430d20.png)

**Comment by FlorianWendelborn** [comment]
Same issue here, I made a bugreport in the other repo already with a video recording: https://github.com/ValveSoftware/Dota2-Gameplay/issues/2720

**Comment by FlorianWendelborn** [comment]
@destinationbugged @DiegoCatalano @stryan @cvlvxi @sskf1990 @tanuj101 @holydrug @ToxyFlog1627 @msaboor35 @snorlax212 @Kostandy can you all check again? They just released an update that fixes this bug for me (I’m on macOS but I had the same symptoms, see comment above)

**Comment by DiegoCatalano** [comment]
It's working now on Ubuntu 22.04

**Comment by holydrug** [comment]
Fixed with new update
Arch linux

**Comment by destinationbugged** [comment]
Yes, works on macOS too! Were done here bois, have a great doto weekend!
